README.md

```diff
@@ -4,6 +4,21 @@ An open source jacketed lab reactor made from off-the-shelf components you can b
 
 Read the motivation behind the project here: [docs/motivation.md](docs/motivation.md)
 
+## Docker Development
+For ease of setup/experimentation we added docker containers for both the GUI and the API.
+
+```bash
+git clone https://github.com/FourThievesVinegar/solderless-microlab.git
+cd solderless-microlab
+
+## To run API & GUI
+docker-compose up --build gui
```

Comment:
I added API as a requirement for GUi so by running GUI it will run APi.  So if we end up having another compose service like a linter/tester it won't also run when we try to run the GUi but with up --build it will run it all

Reply:
In that case line 19 should say "docker-compose up --build api"

Reply:
@0phelia36 haha yeah that was a copy paste error

Reply:
resolved